Reframe README header: nebari-classic positioning (#3222)
nebari-dev/nebari: README.md

> [!NOTE]
> **Nebari is evolving — this repository is now called nebari-classic.**
>

> > Over the years of building and maintaining Nebari, we accumulated a long list of patterns, tooling, and architectural decisions we wanted to apply but were constrained by the current architecture. Rather than continue layering on that foundation, we decided to pause feature development on what we now call **nebari-classic** and invest in a new architecture organized as a thin infrastructure core plus a set of focused software packs:

Author: ```suggestion
> Over the years of building and maintaining Nebari, we accumulated a long list of patterns, tooling, and architectural decisions we wanted to apply but were constrained by the current architecture. Rather than continue layering on that foundation, we decided to pause feature development on what we now call **nebari-classic** and invest in a new architecture organized around the expertise we've built:
```

>
> - [nebari-infrastructure-core (NIC)](https://github.com/nebari-dev/nebari-infrastructure-core) — a composable, modular approach to infrastructure management.

> > - A set of software packs layered on top of NIC — including **Nebi**, **Dask**, and the **data science pack** — that together cover the user-facing functionality nebari-classic provides today. Browse them under [Software Packs](https://nebari.dev/docs/software-packs/) on the docs site, or search the [nebari-dev org for "pack" repositories](https://github.com/orgs/nebari-dev/repositories?q=pack).

Author: ```suggestion
> - A set of software packs layered on top of NIC — including **Nebi** (A newer and more resilient replacement for conda-store), **Dask**, and the **data science pack** (JhubApps, Jupyter* and more) — that together cover the user-facing functionality nebari-classic provides today. Browse them under [Software Packs](https://nebari.dev/docs/software-packs/) on the docs site, or search the [nebari-dev org for "pack" repositories](https://github.com/orgs/nebari-dev/repositories?q=pack).
```

> > - A set of software packs layered on top of NIC — including **Nebi** (A newer and more resilient replacement for conda-store), **Dask**, and the **data science pack** (JhubApps, Jupyter* and more) — that together cover the user-facing functionality nebari-classic provides today. Browse them under [Software Packs](https://nebari.dev/docs/software-packs/) on the docs site, or search the [nebari-dev org for "pack" repositories](https://github.com/orgs/nebari-dev/repositories?q=pack).

Author: ```suggestion
> - A set of component layer "packs" that allow software to be easily integrated on top of NIC through GitOps: including **Nebi** (A newer and more resilient replacement for conda-store), **Dask**, and the **data science pack** (JhubApps, Jupyter* and more). All that together covers the user-facing functionality nebari-classic provides today. Browse them under [Software Packs](https://nebari.dev/docs/software-packs/) on the docs site, or search the [nebari-dev org for "pack" repositories](https://github.com/orgs/nebari-dev/repositories?q=pack).
```

>
> See the NIC design documents in that repository for the full reasoning behind this transition.
>

> > **What this means for nebari-classic.** We will continue to ship critical bug fixes, security and vulnerability patches, and dependency updates on this repository — but at a slower cadence than during active feature development. We expect to keep the maintenance window open until the new architecture is stable and well-tested and operators have had time to migrate, likely through the end of 2026. If that timeline is a problem for your deployment, please reach out at our [community meetings](https://www.nebari.dev/docs/community/) or open a GitHub discussion so we can plan around it.

Author: ```suggestion
> **What this means for nebari-classic.** We will continue to ship critical bug fixes, security and vulnerability patches, and dependency updates on this repository. Though at a slower cadence than during active feature development. We expect to keep the maintenance window open until the new architecture is stable enough and well-tested so that operators have had time to migrate, likely through the end of 2026.
```
